src/frontend/apps/desk/src/features/teams/components/Panel/Panel.tsx

```diff
@@ -11,32 +12,60 @@ export const Panel = () => {
   const { t } = useTranslation();
   const { colorsTokens } = useCunninghamTheme();
 
+  const [isOpen, setIsOpen] = useState(true);
+
   return (
     <Box
-      $width="100%"
+      $width={isOpen ? '100%' : '0'}
       $maxWidth="20rem"
-      $minWidth="14rem"
+      $minWidth={isOpen ? '14rem' : '0'}
       $css={`
-        border-right: 1px solid ${colorsTokens()['primary-300']};
-      `}
+          position: relative;
+          border-right: 1px solid ${colorsTokens()['primary-300']};
+          transition: all 0.2s ease-in-out;
+        `}
```

Comment:
What about adding transition in a variable, and apply this var to all the component with the same transition?

```suggestion
      $css={`
        position: relative;
        border-right: 1px solid ${colorsTokens()['primary-300']};
        transition: ${transition};
      `}
```